infrateam_first_responder.md

```diff
@@ -37,85 +35,85 @@ PRs to this doc are always welcome, but especially while we're ironing things ou
-    * e.g. if Alice and Bob are up next week, `/remind #dlss-infrastructure on Monday at 3 am "@alice is the first responder this week, and @bob is on deck"`
+  * Documentation on Slack's `/remind` command:  https://get.slack.help/hc/en-us/articles/208423427-Set-a-reminder
+    * E.g., if Alice and Bob are up next week, `/remind #dlss-infrastructure on Monday at 3 am "@alice is the first responder this week, and @bob is on deck"`
 
-## Sign Up for Your Next First Responder Shift
+### Sign Up for Your Next First Responder Shift
 
 The infrastructure team has 8 developers, so you should be taking a shift every 8 or so weeks.
-* schedule:  https://docs.google.com/spreadsheets/u/1/d/13TJR93Yc9_eF5B7w4XDx6ggG_wb3aLkgCHjpLwmHPBA/
+* Schedule:  https://docs.google.com/spreadsheets/u/1/d/13TJR93Yc9_eF5B7w4XDx6ggG_wb3aLkgCHjpLwmHPBA/
 
-## Proactively Check for Production Problems
+### Proactively Check for Production Problems
 
 See [How to Proactively Check for Production Problems](#how-to-proactively-check-for-production-problems) section below for specifics.
 
-## Triage Production Problems
+### Triage Production Problems
 
-If a user reports a problem, or if one is surfaced from monitoring, the first responder is meant to timebox an investigation of the problem (_TBD: 30 min?_).  See [How to Triage Production Problems](#how-to-triage-production-problems) section below for more specifics.
+If a user reports a problem, or if one is surfaced from monitoring, the first responder ought to timebox an investigation of the problem (_TBD: 30 min?_). See [How to Triage Production Problems](#how-to-triage-production-problems) section below for more specifics.
 
-## Improve Troubleshooting Documentation as Needed
+### Improve Troubleshooting Documentation as Needed
 
-If you need to triage or troubleshoot a problem and realize some documentation is missing, please provide it.  List of appropriate places:
-* https://github.com/sul-dlss/DevOpsDocs - e.g. what an ops or devops person would need to know to handle the situation
+If you need to triage or troubleshoot a problem and realize some documentation is missing, please provide it. List of appropriate places:
+* https://github.com/sul-dlss/DevOpsDocs - e.g., what an ops or devops person would need to know to handle the situation
 * README or other top level markdown doc in codebase (viewable via github)
   * Which codebase would need to be apparent from the problem
 * Wiki for the codebase
-* ? - in general, just try to consider where the person interested in the info might look first.  A user might go to the wiki, a dev might go to the README, ops folks might head to DevOpsDocs.  Use your best judgement and ask for feedback if unsure.
+* ? - in general, just try to consider where the person interested in the info might look first. A user might go to the wiki, a dev might go to the README, ops folks might head to DevOpsDocs. Use your best judgement and ask for feedback if unsure.
```

Comment:
might as well get rid of "just" while you're at it.

Reply:
done.